CMS/translations.py

```diff
@@ -702,8 +702,8 @@
         'cy': 'Chwiliad newydd'
     },
     'no_courses_found': {
-        'en': 'No courses found - remove or change some filters',
-        'cy': 'Ni chafwyd hyd i unrhyw cyrsiau - dileu neu newid rhai hidlwyr'
+        'en': 'No courses found, remove or change some filters',
```

Comment:
Here, a comma is used instead of a hyphen - in my opinion this is incorrect grammar